### ISSUE TITLE
Constructors don't get added to refmap

### ISSUE THREAD

**Issue by killjoy1221** [SpongePowered/Mixin#97]
The name <init> may not change, but the parameters do. Specifically when using notch names.

I'm applying a mixin to `<init>(Lnet/minecraft/client/renderer/entity/RenderManager;Z)V`, but it fails when in an obfuscated environment because the description becomes `<init>(Lcpt;Z)V`

The issue goes away after I manually add it to the refmap.

I assume the same happens with other non-obfuscated methods that use obfuscated parameters.

**Comment by Mumfrey** [comment]
I know about this and have been meaning to sort it for a while and just haven't gotten around to it. I'll roll up a fix with some other obf changes today.

**Comment by killjoy1221** [comment]
Now they get added, but the descriptor doesn't get remapped.

`"<init>(Lnet/minecraft/client/renderer/entity/RenderManager;Z)V": "Lctc;<init>(Lnet/minecraft/client/renderer/entity/RenderManager;Z)V"` gets added to the notch data.

**Comment by Minecrell** [comment]
@killjoy1221 I think this could be possibly caused by the fix for #99 (see https://github.com/SpongePowered/Mixin/issues/99#issuecomment-175132797). Can you try using a build before that commit (e.g. `0.4.17-20160126.141141-3`).

With Gradle 2.4+ you can use it like this:
```gradle
dependencies {
    compile 'org.spongepowered:mixin:0.4.17-20160126.141141-3'
}
```

**Comment by killjoy1221** [comment]
Yeah, that was it.
